doc/source/how-to/supporting-python-versions.rst

```diff
@@ -8,22 +9,22 @@ more information, see `Status of Python branches
 +---------+------------+-------------+-----------------------+--------+
 | Version | PEP        | Released    | Security Support Ends | Status |
 +---------+------------+-------------+-----------------------+--------+
-| 3.11    | `PEP 664`_ | 03 Oct 2022 | 03 Oct 2027           | Dev    |
+| 3.12    | `PEP 693`_ | 02 Oct 2023 |    Oct 2028           | Dev    |
 +---------+------------+-------------+-----------------------+--------+
-| 3.10    | `PEP 619`_ | 04 Oct 2021 | 04 Oct 2026           | Dev    |
+| 3.11    | `PEP 664`_ | 03 Oct 2022 |    Oct 2027           | Dev    |
 +---------+------------+-------------+-----------------------+--------+
-| 3.9     | `PEP 596`_ | 05 Oct 2020 | 05 Oct 2025           | Stable |
+| 3.10    | `PEP 619`_ | 04 Oct 2021 |    Oct 2026           | Stable |
 +---------+------------+-------------+-----------------------+--------+
-| 3.8     | `PEP 569`_ | 14 Oct 2019 | 14 Oct 2024           | Stable |
+| 3.9     | `PEP 596`_ | 05 Oct 2020 |    Oct 2025           | Stable |
 +---------+------------+-------------+-----------------------+--------+
-| 3.7     | `PEP 537`_ | 27 Jun 2018 | 27 Jun 2023           | Stable |
+| 3.8     | `PEP 569`_ | 14 Oct 2019 |    Oct 2024           | Stable |
 +---------+------------+-------------+-----------------------+--------+
```

Comment:
3.13 is on dev mode. 3.12 is already stable. 3.8 should be dropped from this list since we are no longer supporting it.